### DISCUSSION TITLE
Support for radio (audio only) streams

### DISCUSSION THREAD

**Discussion by pablohoffman** [streamlink/streamlink#5211] [Q&A] [answered]
### Description

Are there plans to support audio-only streams in streamlink?. I'm thinking on streaming net radio streams, as opposed to video streams.

If this is out of scope for streamlink, does anyone know of an alternative for radio (audio-only) streams?

Thanks!

**Comment by bastimeyer** [comment, marked as answer]
https://github.com/streamlink/streamlink/blob/master/CONTRIBUTING.md#plugin-requests

> 7. Sites which don't provide any video streaming content, eg. radio stations

It doesn't matter which kind of content a stream includes, but we're not going to add any (new) plugins for audio-only streams.

If you have a URL for an audio-only stream, then you can use the streaming-protocol plugins directly:
https://streamlink.github.io/cli/protocols.html

Otherwise, write your own plugin which extracts the stream URL from a website.
